collections/climate-change-initiative.yaml

```diff
@@ -0,0 +1,84 @@
+        - Name: Description
+          Title: Description
+      Rows:
+        - ECV: Aerosol
+          Description: Set of global aerosol ECV products from different sensors
+        - ECV: Cloud
+          Description: Long-term coherent cloud property data sets
+        - ECV: Greenhouse Gases
+          Description: Carbon Dioxide and Methane products
+        - ECV: Ozone
+          Description: Ozone products from ESA and Third Party Missions
+        - ECV: Ocean Colour
+          Description: Chlorophyll, Inherent Optical Properties, and Water-leaving Reflecatnce products
+        - ECV: Sea Ice
+          Description: Sea ice Concentration and Sea ice Thickness products
+        - ECV: Sea Level
+          Description: Sea Level ECV products from different instruments
+        - ECV: Sea State
+          Description: Wave height products integrating different sensors
+        - ECV: Sea Surface Salinity
+          Description: Sea Surface Salinity products 
+        - ECV: Sea Surface Temperature
+          Description: Sea Surface Temperature products
+        - ECV: Water Vapour
+          Description: Products on Water Vapour content in the Atmosphere     
+        - ECV: Antarctic Ice Sheet
+          Description: Location, Mass Balance, and Velocities of the Antarctic Ice Sheet
+        - ECV: Above-Ground Biomass
+          Description: Global Datasets for forest Above-Ground Biomass 
+        - ECV: Fire
+          Description: Different Burned Area products
+        - ECV: Glaciers
+          Description: Inventory of Glacier Outlines  
+        - ECV: Greenland Ice Sheet
+          Description: Different products characterizing the dynamics of Greenland Ice Sheet
+        - ECV: Lakes
+          Description: Different variables characterizing Lakes on a global scale
+        - ECV: Land Cover
+          Description: Global classification of Land Cover and a Water Bodies Map
+        - ECV: Land Surface Temperature
+          Description: Land Surface Temperature products integrating different sensors 
+        - ECV: Permafrost
+          Description: Active Layer Thickness, Ground Temperature, and Extent products for the Northern Hemisphere
+        - ECV: Snow
+          Description: Long time-series of daily Snow Water Equivalent and Snow Cover Fraction
+        - ECV: Soil Moisture
+          Description: Soil Moisture products from different sensors   
+          
+Contact: "[CCI Open Data Portal Helpdesk](https://climate.esa.int/en/helpdesk/)"
+Provider: "[ESA Climate Office](https://climate.esa.int/en/esa-climate/esa-climate-office/)"
+ManagedBy: "[Brockmann Consult](https://www.brockmann-consult.de/)"
+Tags:
+    - Climate
+    - CCI
+    - open data
+    - xcube
+    - raster
+    - essential climate variables
+License: |
+  ESA CCI Data Policy: free and open access ([details](https://climate.esa.int/sites/default/files/CCI_Data_Policy_v1.1.pdf)).
+Resources:
+ - Group: xcube Resources
+   StoreID: cciodp

```

Comment:
The handler does not recognise `StoreID`. Please refer to the metadata table in the [contributions file](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md):

| Field | Type | Description & Style |
| --- | --- | --- |
|**Resources > DatasetName**|String|Name of the xcube dataset.|
|**Resources > StoreInstanceID**|String|ID of the xcube store.|
|**Resources > StoreTitle**|String|Name of xcube store of the collection.|
|**Resources > DataID**|String|Name identifier of the collection in the xcube store.|
|**Resources > Notes**|MD| More info regarding the collection.|

The initial setup of these fields was developed with the help of the provided table in [this gitext issue](https://gitext.sinergise.com/dcfs/common/-/issues/286#note_35253). Maybe this helps. 🙂 

Current view:
![image](https://user-images.githubusercontent.com/60575896/205029210-215cda56-c6dc-45b5-8c34-cf946db8d350.png)